src/_tmap/cereal/types/tuple.hpp

```diff
@@ -95,7 +95,7 @@ namespace cereal
       template <class Archive, class ... Types> inline
       static void apply( Archive & ar, std::tuple<Types...> & tuple )
       {
-        serialize<Height - 1>::template apply( ar, tuple );
+        serialize<Height - 1>::template apply<>( ar, tuple );
```

Comment:
It is always better to put more information that "Appease clang" for changes like this.

In future I would like to see the specific "appeasement" that is occurring but in this case I assume it is a warning about template ambiguity.